redis/src/aio/connection_manager.rs

```diff
@@ -18,6 +18,82 @@ use std::sync::Arc;
 use tokio_retry::strategy::{jitter, ExponentialBackoff};
 use tokio_retry::Retry;
 
+/// Apply a maximum delay. No retry delay will be longer than this `max_delay`.
+#[derive(Clone, Debug, Default)]
+pub struct ConnectionManagerConfig {
+    /// The resulting duration is calculated by taking the base to the `n`-th power,
```

Comment:
since these are private fields, this documentation should either be moved or copied to the relevant set_foo functions - ATM the user won't see these docs.